typing/HACKING.adoc

```diff
@@ -3,7 +3,7 @@ will need a good understanding of the OCaml type system and type
 inference. Here is a reading list to ease your discovery of the
 typechecker:
 
-http://caml.inria.fr/pub/docs/u3-ocaml/index.html[Using, Understanding, and Unraveling the OCaml Language by Didier Rémy] ::
+http://caml.inria.fr/pub/docs/u3-ocaml/index.html[Using, Understanding, and Unraveling the OCaml Language by Didier R&eacute;my] ::
```

Comment:
Funny fact: I've known Didier for more than 25 years, and I learned two weeks ago that his name officially doesn't have an accent...